pymongo/auth.py

```diff
@@ -99,9 +104,25 @@ def __hash__(self):
 """Mechanism properties for MONGODB-AWS authentication."""
 
 
+_OIDCProperties = namedtuple(
```

Comment:
Use an untyped data class here to be consistent with _OIDCCache?

Reply:
Done